packages/common-db/src/lib/common/session.ts

```diff
@@ -59,21 +66,22 @@ export function createAdapterStore(session: Session, defaultTtl = 3600): any {
          * @param sess the session
          * @param fn callback
          */
-        set(sid: string, ttl: number, sess: Session, fn: (args: any) => void): void {
+        set(
+            sid: string,
+            ttl: number | SessionData,
+            sess: SessionData | ((err?: Error | null) => void),
+            fn: (err?: Error | null) => void,
+        ): void {
             if (typeof ttl === 'object') {
-                fn = sess;
+                fn = sess as (err?: Error | null) => void;
                 sess = ttl;
                 // analyse if the session is stored directly from express session
-                ttl =
-                    sess && sess.cookie && sess.cookie.originalMaxAge
-                        ? Math.round(sess.cookie.originalMaxAge / 1000)
-                        : defaultTtl;
+                ttl = sess?.cookie?.originalMaxAge ? Math.round(sess.cookie.originalMaxAge / 1000) : defaultTtl;
             }
             ttl = ttl || defaultTtl;
-            this.adapter.setSession(sid, ttl, sess, function () {
+            this.adapter.setSession(sid, ttl, sess, function (err?: Error | null): void {
```

Comment:
maybe use lambda function here?

Reply:
No. Because we do not want to give the 'this' context in callback